src/system_identity/environment.rs

```diff
@@ -0,0 +1,83 @@
+use std::sync::LazyLock;
+
+use http::Uri;
+
+// Known endpoints. Should these be configurable instead? Otherwise,
+// if they change we will need to create a new release.
+// Staging endpoints
```

Comment:
Good Point! I was thinking the same. However, if the library (and the cli) are supposed to ease the System Identity management, keeping them there makes perfect sense.

Maybe some sort of integration test to assure that the endpoints are correct would be useful (out of scope of this PR)